src/styles/Menu.module.css

```diff
@@ -0,0 +1,19 @@
+.list {
+  background-color:blue;
+  text-align: center;
+  padding: var(--base-unit);
+}
+
+.item {
+  width: 100%;
+  padding: var(--base-unit);
```

Comment:
The links inside the menu do not cover the whole button, how about removing the `padding` from here and creating a different style for the items to cover the whole area?

```css
.itemLink {
  display: block;
  padding: var(--base-unit);
}
```

Of course, it will need to be used in `main_menu.tsx`:

```tsx
function LinkEntry({ href, label }: LinkEntryProps) {
  return (
    <li className={styles.item}>
      <Link className={styles.itemLink} href={href}>
        {label}
      </Link>
    </li>
  );
}
```